congratulations/Congratulations.py

```diff
@@ -23,29 +24,30 @@ def __init__(self):
         self.challenge_duration = 100        # Set this manually for now. Pass it in as an argument later.
         self.challenge_hashtag = "#100DaysOfCode"
 
-        CANVAS_WIDTH = 300
-        CANVAS_HEIGHT = 250
-        BTTN_WIDTH = 12
-        self.canvas = Tk.Canvas(self.root, width = CANVAS_WIDTH, height = CANVAS_HEIGHT)
-        self.canvas.pack()
+        CANVAS_WIDTH = 270
+        CANVAS_HEIGHT = 160
+        BTTN_WIDTH = 150
+        #commented canvas decleration out, I don't understand why it's even here
```

Comment:
You are right, that is a mistake, since this window is created as a toplevel window. I plan on standardizing the way we create windows in the future, but for now we just need to remove these lines that you have commented out